array-destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,11 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPrice: 1.0 },
   { itemName: "Hash Brown", quantity: 4, unitPrice: 0.4 },
 ];
+let total = 0;
+console.log('QTY\tITEM \t\t\t TOTAL');
+order.filter(({quantity,itemName,unitPrice}=order)=>{
+console.log(`${quantity}\t${itemName.padEnd(20)}\t${unitPrice.toFixed(2)}`);
+  let sum = unitPrice * quantity;
+    total+= sum;
```

Comment:
Keeping the running total this way is reasonable, and we do get the correct total in the end. Maybe we can use this `sum` in our "Total" column, though?